ci-operator/step-registry/openshift/e2e/test/pausedworkers/openshift-e2e-test-pausedworkers-commands.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred
+export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.json
+export GCP_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/gce.json
+export HOME=/tmp/home
+export PATH=/usr/libexec/origin:$PATH
+
+if [[ -n "${TEST_CSI_DRIVER_MANIFEST}" ]]; then
+    export TEST_CSI_DRIVER_FILES=${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}
+fi
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+mkdir -p "${HOME}"
+
+# Override the upstream docker.io registry due to issues with rate limiting
+# https://bugzilla.redhat.com/show_bug.cgi?id=1895107
+# sjenning: TODO: use of personal repo is temporary; should find long term location for these mirrored images
+export KUBE_TEST_REPO_LIST=${HOME}/repo_list.yaml
+cat <<EOF > ${KUBE_TEST_REPO_LIST}
+dockerLibraryRegistry: quay.io/sjenning
```

Comment:
Unrelated: I think mirroring images is resolved upstream.

Reply:
I don't want to diverge from `openshift-e2e-test` too much :/